install.rb

```diff
@@ -356,6 +369,7 @@ def prepare_installation
   InstallOptions.lib_dir = libdir
   InstallOptions.man_dir = mandir
   InstallOptions.var_dir = vardir
+  InstallOptions.publicdir = publicdir
```

Comment:
I'd change this to `InstallOptions.public_dir` just because the others follow the same convention. Well, except for `codedir`.

Reply:
Not sure if this change alone has any impact on the functionality but I agree on the change.